### ISSUE TITLE
Is sscep lib compliant with SCEP RFC8894? 

### ISSUE THREAD

**Issue by ml16685** [certnanny/sscep#129]
Hello,

Could you please clarify, whether sscep lib is compliant with the latest SCEP RFC8894:
https://datatracker.ietf.org/doc/html/rfc8894 ?

If not, then on which internet draft sscep lib is currently basing on and are  there plans to support RFC8894?

Thanks.

**Comment by gotthardp** [comment]
Hello. We don't claim to be compliant to any particular SCEP specification. The project relies on contributions from people who usually address interoperability with individual SCEP servers. It would be great if you could help us assessing compliance with the mandatory requirements of RFC 8894.

**Comment by ml16685** [comment]
Thank you for the info.

>>> It would be great if you could help us assessing compliance with the mandatory requirements of RFC 8894.
I have too few knowledge in SCEP for such a complicated task.

**Comment by mbartosch** [comment]
Let me clarify: you would like to get a formal confirmation of standard compliance, you are aware that this is a complicated task and you would like to get this service for free?

**Comment by ml16685** [comment]
> Let me clarify: you would like to get a formal confirmation of standard compliance, you are aware that this is a complicated task and you would like to get this service for free?

No, I just wanted to clarify if the lib is claimed to be RFC 8894 compliant. As I understood from the first response, it is not.
